### PR TITLE
added mean aggregation over each metric

### PR DESCRIPTION
**Pull request by dzhovi**
https://github.com/yegor256/cam/issues/398

### CODE REVIEW

**Comment by dzhovi** [comment]
@yegor256 could you check logs of a failed job?

File: steps/aggregation-functions/mean.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+mkdir -p "${output_folder}"
+
+sum=0
+count=0
+
+values=$(awk -F, 'NR > 1 {print $3}' "${metric_file}")
+
+while IFS= read -r value; do
+    sum=$(echo "$sum + $value" | bc)
+
```
Comment (yegor256) [inline steps/aggregation-functions/mean.sh:39]: @dzhovi better avoid empty lines inside code blocks: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html

File: tests/steps/aggregation-functions/test-mean.sh [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+stdout=$2
+
+{
+    dir="${TARGET}"
+    mkdir -p "${dir}"
+    touch "${dir}/LCOM5.csv"
+    echo "repo,java_file,LCOM5" > "${dir}/LCOM5.csv"
+    echo "kek,src/main/kek,42.0000" >> "${dir}/LCOM5.csv"
+
```
Comment (yegor256) [inline tests/steps/aggregation-functions/test-mean.sh:34]: @dzhovi same here, empty lines are not welcome inside code blocks

**Review by yegor256 — CHANGES_REQUESTED** [review]
@dzhovi looks very good! see a few comments above

**Comment by dzhovi** [comment]
@yegor256 could you check the log of a failed job? There is some problem with signatures

File: steps/aggregation-functions/mean.sh (hunk 1 above)
Comment (dzhovi) [reply steps/aggregation-functions/mean.sh]: @yegor256 fixed

File: tests/steps/aggregation-functions/test-mean.sh (hunk 2 above)
Comment (dzhovi) [reply tests/steps/aggregation-functions/test-mean.sh]: @yegor256 fixed

**Comment by dzhovi** [comment]
@yegor256 i tried to change the mirror, but apparently it is prohibited to do(git tells me that it creates conflict)

**Comment by yegor256** [comment]
@dzhovi looks good, thanks!

**Pull request merged**